addons/common/functions/fnc_addItemsToArsenal.sqf

```diff
@@ -0,0 +1,33 @@
+#include "script_component.hpp"
+/*
+ * Author: 3Mydlo3
+ * Function adds items to given arsenal.
+ *
+ * Arguments:
+ * 0: Object with arsenal <OBJECT>
+ * 1: Item or list of items to add <STRING/ARRAY>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [player, "ItemGPS"] call afsk_common_fnc_addItemsToArsenal
+ *
+ * Public: No
+ */
+
+params ["_object", "_items"];
+
```

Comment:
Do:
```sqf
if (_items isEqualType "") then {
    _items = [_items];
};
```
this will reduce nesting in further `if`s

Reply:
Yes, that's much better.